### PR TITLE
telegram-notify 1.0.1

### PR DESCRIPTION
**Pull request by mcaldas**
TagCheck

### New Pull Request Checklist

*Please mark the points which you did / accept.*

- [x] __I will not move an already shared step version's tag to another commit__
- [x] I read the [Step Development Guideline](https://github.com/bitrise-io/bitrise/blob/master/_docs/step-development-guideline.md)
- [x] I have a test for my Step, which can be run with `bitrise run test` (in the step's repository)
- [x] I did run `bitrise run audit-this-step` (in the step's repository - note, if you don't have this workflow in your `bitrise.yml`, [you can copy it from the step template](https://github.com/bitrise-steplib/step-template/blob/master/bitrise.yml).)
- [x] I read and accept the [Abandoned Step policy](https://github.com/bitrise-io/bitrise-steplib#abandoned-step-policy)

### CODE REVIEW

File: steps/telegram-notify/1.0.1/step.yml [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+  bash:
+    entry_file: step.sh
+deps:
+  brew:
+  - name: curl
+  apt_get:
+  - name: curl
+is_requires_admin_user: true
+is_always_run: true
+is_skippable: false
```
Comment (koral--) [inline steps/telegram-notify/1.0.1/step.yml]: Steps like this usually should be skippable. Look at slack step as a reference: https://github.com/bitrise-io/steps-slack-message/blob/master/step.yml#L22
Docs: https://devcenter.bitrise.io/tips-and-tricks/dont-mark-build-failed-if-step-fails/

File: steps/telegram-notify/1.0.1/step.yml [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+run_if: ""
+inputs:
+- opts:
+    description: |
+      The bot token.
+      Can also be taken from env var `TELEGRAM_BOT_TOKEN`
+    is_required: true
+    title: Telegram Bot Token
+  telegram_bot_token: $TELEGRAM_BOT_TOKEN
```
Comment (koral--) [inline steps/telegram-notify/1.0.1/step.yml]: Inputs like tokens should be usually sensitive: https://devcenter.bitrise.io/builds/sensitive-input-field/

File: steps/telegram-notify/1.0.1/step.yml (hunk 1 above)
Comment (trapacska) [reply steps/telegram-notify/1.0.1/step.yml]: 👍

File: steps/telegram-notify/1.0.1/step.yml (hunk 2 above)
Comment (trapacska) [reply steps/telegram-notify/1.0.1/step.yml]: 👍 good catch

**Comment by trapacska** [comment]
@mcaldas please close this pr, remove the branch of this PR in your fork and re-run the share.

**Pull request closed without merging**